packages/cozy-harvest-lib/src/components/AccountLogoutButton.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import omit from 'lodash/omit'
+
+import { Button } from 'cozy-ui/react/Button'
+import { translate } from 'cozy-ui/react/I18n'
+import { withMutations } from 'cozy-client'
+
+import accountsMutations from '../connections/accounts'
+
+export class AccountLogoutButton extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loggingOut: false
```

Comment:
Instead of a flag I suggest to use a status between: `IDLE`, `PENDING` (or `DELETING`), `ERRORED`.

Reply:
I disagree, we don't have different state to handle here, I am not comfortable to use many strings just for this simple case